remme/tp/pub_key.py

```diff
@@ -81,26 +118,27 @@ def _store_pub_key(self, context, signer_pubkey, transaction_payload):
             LOGGER.debug(f'entity_hash {transaction_payload.entity_hash}')
             raise InvalidTransaction('Entity hash or signature not a hex format')
 
-        # FIXME: For support PKCS1v15 and PSS
-        LOGGER.warn('HAZARD: Detecting padding for verification')
-        sigerr = 0
-        pkcs = padding.PKCS1v15()
-        pss = padding.PSS(mgf=padding.MGF1(hashes.SHA512()), salt_length=padding.PSS.MAX_LENGTH)
-        for _padding in (pkcs, pss):
-            try:
-                cert_signer_pubkey.verify(ehs_bytes, eh_bytes, _padding, hashes.SHA512())
-                LOGGER.warn('HAZARD: Padding found: %s', _padding.name)
-            except InvalidSignature:
-                sigerr += 1
-
-        if sigerr == 2:
+        try:
+            if transaction_payload.public_key_type == NewPubKeyPayload.RSA:
+                self._verify_rsa(ehs_bytes, eh_bytes, loaded_public_key,
+                                 transaction_payload.rsa_signature_padding)
+
+            # TODO: Add support in future
```

Comment:
Why is that disabled?

Reply:
@eugene-babichenko implementation postponed on JS libriary

Reply:
The make the TODO more descriptive.

Reply:
@eugene-babichenko will do